srml/babe/src/lib.rs

```diff
@@ -116,7 +116,13 @@ decl_storage! {
 		LastTimestamp get(last): T::Moment;
 
 		/// The current authorities set.
-		Authorities get(authorities): Vec<AuthorityId>;
+		Authorities get(authorities) build(|config: &GenesisConfig| {
```

Comment:
Here you can do the same as @andresilva has written below.

Reply:
Thank you.